rust/src/coordinator.rs

```diff
@@ -525,7 +528,8 @@ impl Coordinator {
         info!("going to update phase");
         self.freeze_sum_dict();
         self.phase = Phase::Update;
-        self.emit_event(ProtocolEvent::StartUpdate(self.sum_dict.clone()));
+        let scalar = 1_f64 / (self.expected_participants as f64 * self.update);
```

Comment:
i decided against making the `scalar` part of the general round parameters, because it is only of use to update participants and it is only used during the composition of the update message. it could in theory be part of the general round parameters because its value is known at the beginning of a round though.

i suggest that we keep it as part of the update phase data for now, and once the extension to the pet protocol for arbitrary scalars is going to be implemented, this scalar is not going to be communicated at all.